subcommands/version/cmd.go

```diff
@@ -38,7 +38,7 @@ func doVersion(cmd *cobra.Command, args []string) {
 		}
 		if err := update.Do(); err != nil {
 			if os.IsPermission(err) {
-				DieNotNil(fmt.Errorf("there was a permission error while updating fioctl: %w. Please run again as an admin or root.", err))
+				DieNotNil(fmt.Errorf("there was a permission error while updating Fioctl: %w. Please run again as an admin or root.", err))
```

Comment:
I've noticed that you make the first capital letter for all error messages.
Let's do the same here then.